complete/src/main/java/hello/repositories/UsersRepository.java

```diff
@@ -0,0 +1,11 @@
+package hello.repositories;
+
+import org.springframework.data.repository.CrudRepository;
+
+import hello.models.User;
+
+// This will be AUTO IMPLEMENTED by Spring into a Bean called usersRepository
+// CRUD refers Create, Read, Update, Delete
+public interface UsersRepository extends CrudRepository<User, Long> {
```

Comment:
I think `UserRepository` is more conventional (refer to the entity as singular).